tests/base/src/test/java/org/wildfly/security/auth/FileSystemSecurityRealmTest.java

```diff
@@ -114,13 +131,14 @@ public void testCreateIdentityWithLevels() throws Exception {
         identity.dispose();
     }
 
-    @Test
-    public void testCreateIdentityWithLevelsEncryption() throws Exception {
+   @Test
+    public void testCreateIdentityWithLevelsIntegrity() throws Exception {
```

Comment:
Just to check because it's a bit hard to tell from the diff, was the encryption test removed?

Reply:
It doesn't look like the tests validate the signature that's written to the identity file. That would be good to add.

Reply:
I think the encryption test got moved around, Ive put them all together now